### PR TITLE
Front appointment details

### PR DESCRIPTION
**Pull request by aksuz**

### CODE REVIEW

File: SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<head>
```
Comment (bdettlaff) [inline SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.html]: I think that this head elements are not necessary

File: SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    patientUUID: any;
+
+    constructor(private appointmentService: AppointmentDetailsService, private route: ActivatedRoute) {
+
+    }
+
+    ngOnInit(): void {
+        this.patientUUID = this.route.snapshot.paramMap.get('patientUUID');
+        this.getAppointment();
+        //53579dae-b861-4186-826a-dc534d66e44e UUID to test component
```
Comment (bdettlaff) [inline SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts]: This comment should be removed

File: SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+              'PLANNED',
+              '15-05-2021',
+              '8:30',
+              'Lodz',
+              '12-345',
+              'Streeeeeeet 11/223',
+              'Przychodnia XYZ');*/
+    }
+
+    getAppointment(): void {
```
Comment (bdettlaff) [inline SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts:25]: On this method you should use ctrl+alt+l. Moreover, console.log with this.appoitment should be removed.

File: SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.html (hunk 1 above)
Comment (aksuz) [reply SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.html]: deleted

File: SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts (hunk 2 above)
Comment (aksuz) [reply SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts]: done

File: SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts (hunk 3 above)
Comment (aksuz) [reply SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts:25]: done

File: SzczepionkaWEB/src/app/service/appointment-details/appointment-details.service.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+export class AppointmentDetailsService {
+
+
+    constructor(private http: HttpClient) {
+    }
+
+    getAppointment(patientUUID: string): Observable<AppointmentDetails> {
+        return this.http.get<AppointmentDetails>('http://localhost:8080/appointment/' + patientUUID, httpOptions);
+    }
+
```
Comment (xNIGHTHAWKx) [inline SzczepionkaWEB/src/app/service/appointment-details/appointment-details.service.ts]: Redundant spacing.

File: SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
+.container {
```
Comment (xNIGHTHAWKx) [inline SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.css:1]: This is my fav css in whole project 🥇

File: SzczepionkaWEB/src/app/service/appointment-details/appointment-details.service.ts (hunk 4 above)
Comment (aksuz) [reply SzczepionkaWEB/src/app/service/appointment-details/appointment-details.service.ts]: fixed

**Review by bdettlaff — APPROVED** [review]

**Pull request merged**